### PR TITLE
Adds script for ship hero login

### PR DESCRIPTION
**Pull request by iamplugged**
Issue: https://github.com/Screenly/playground/issues/14

- Adds shiphero-signin-via-credentials.js login script for shiphero: [shipping.shiphero.com](https://shipping.shiphero.com/)

### CODE REVIEW

**Comment by vpetersson** [comment]
@iamplugged Please also add this to the README.

File: javascript-injectors/shiphero-signin-via-credentials.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      changeValue(emailEl, email)
+      changeValue(passwordEl, password)
+
+      document.querySelector('button[name="submit"]').click()
+    } catch (error) {
+      console.log(error)
+    }
+  }
+
+  setTimeout(login, 5000)
```
Comment (rusko124) [inline javascript-injectors/shiphero-signin-via-credentials.js:31]:
```suggestion
(function () {
  const email = '<YOUR_EMAIL>'
  const password = '<YOUR_PASSWORD>'
  const authLocation = '/some/signin/page'
  
  function setValue (selector, value) {
    const element = document.querySelector(selector)
    element.value = value;
    element.dispatchEvent(new Event('change'));
  }
  
  function submitForm () {
      document.querySelector('button[name="submit"]').click()
  }
  
  function login () {
    try {
      setValue('input[name="email"]', email)
      setValue('input[name="password"]'', password)
      submitForm()
    } catch (error) {
      console.warn(error)
      setTimeout(login, 1000)
    }
  }

  if (window.location.pathname === authLocation) {
      login()
  }
```

**Review by rusko124 — CHANGES_REQUESTED** [review]

File: javascript-injectors/shiphero-signin-via-credentials.js (hunk 1 above)
Comment (iamplugged) [reply javascript-injectors/shiphero-signin-via-credentials.js:31]: @rusko124 

This snippet does not work for Ship Hero. It seems to be React based, so normal events don't seem to work.

`const element = document.querySelector(selector)
    element.value = value;
    element.dispatchEvent(new Event('change'));`

Comment (vpetersson) [reply javascript-injectors/shiphero-signin-via-credentials.js]: This is kinda weird to me but OK, let's use the recommendation.

Comment (iamplugged) [reply javascript-injectors/shiphero-signin-via-credentials.js]: Updated to fix pending lint issues

**Review by rusko124 — COMMENTED** [review]
LGTM, but I think we should run the script on a real device before landing

File: javascript-injectors/README.md [hunk 2]
```diff
@@ -1,18 +1,20 @@
...
 Screenly's JavaScript Injector feature allows users to perform automation on websites, such as:
 
-* Logging into websites using credentials (or a cookie)
+* Logging into sites using credentials (or a cookie)
 * Close modals, such as GDPR consent dialogues
 * Scroll down a page
 
 ## Usage
 
-To use the JavaScript Injector, you need first to create the web asset. You can do that in the [user interface](https://login.screenlyapp.com) or [using the API](https://developer.screenlyapp.com/#operation/assets_create). (While you could include the JavaScript directly in the asset creation, it can be beneficial to do the JavaScript snippet as a PATCH as it allows you to update your JavaScript code easily.)
```
Comment (vpetersson) [inline javascript-injectors/README.md:13]: Why was this removed?

Comment (iamplugged) [reply javascript-injectors/README.md:13]: Not removed actually. Just a newline to fix a lint issue saying a very long line with some 412 characters.

**Review by vpetersson — APPROVED** [review]

**Comment by iamplugged** [comment]
@rusko124 Are we good to merge it? Need your approval.

**Comment by iamplugged** [comment]
@rusko124 Awaiting your approval on this.

**Review by rusko124 — APPROVED** [review]

**Pull request merged**